CleverPush/CleverPush/Source/CPTopicDialogCell.m

```diff
@@ -10,4 +12,29 @@ - (void)setSelected:(BOOL)selected animated:(BOOL)animated {
     [super setSelected:selected animated:animated];
 }
 
+-(void)createSeparator{
+    UIView *separatorView = [self.contentView viewWithTag:separatorTag];
+    if (separatorView == nil) {
+        UIView *separatorView = [[UIView alloc] initWithFrame:CGRectMake(8, self.bounds.size.height-1, (self.bounds.size.width - 16), 0.45)];
+        separatorView.tag = separatorTag;
+        [separatorView setBackgroundColor:[UIColor lightGrayColor]];
+        [self.contentView addSubview:separatorView];
+    }
+}
+
+-(void)updateSeparatorWithTopicHighlighter :(BOOL) isTopicHighlighter{
```

Comment:
Code style (space missing):
`- (void).....`

Code style: no space before `:(BOOL)` and before `isTopicHighlighter`